src/EnergyPlus/PluginManager.cc

```diff
@@ -688,12 +718,13 @@ void PluginInstance::reportPythonError([[maybe_unused]] EnergyPlusData &state)
 
         // traceback.format_exception returns a list, so iterate on that
         if (!pyth_val || !PyList_Check(pyth_val)) { // NOLINT(hicpp-signed-bitwise)
-            EnergyPlus::ShowFatalError(state, "In reportPythonError(), traceback.format_exception did not return a list.");
+            EnergyPlus::ShowContinueError(state, "In reportPythonError(), traceback.format_exception did not return a list.");
+            return;
         }
 
         unsigned long numVals = PyList_Size(pyth_val);
         if (numVals == 0) {
-            EnergyPlus::ShowFatalError(state, "No traceback available");
+            EnergyPlus::ShowContinueError(state, "No traceback available");
```

Comment:
We don't need to throw fatal errors in here, just issue continuations and return, the code is already destined for a more meaningful fatal error at this point.